### PR TITLE
Reduce verbose output

### PR DESCRIPTION
**Pull request by DreierF**
- [ ] [No new Teamscale findings](https://build.cqse.eu/teamscale/delta.html#input/jacoco-client/?showMergeFindings=true)
- [ ] [Changes are tested adequately](https://build.cqse.eu/teamscale/tests.html#/jacoco-client/)
- [ ] README.md updated in case of user-visible changes
- [ ] CHANGELOG.md updated
- [ ] build.gradle version number increased according to semantic versioning

### CODE REVIEW

File: report-generator/src/main/java/com/teamscale/report/testwise/jacoco/cache/ClassNotFoundLogger.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	}
+
+	/** Saves the given class to be logged later on. Ensures that the class is only logged once. */
+	/* package */ void log(String fullyQualifiedClassName) {
+		if (!alreadyLoggedClasses.contains(fullyQualifiedClassName)) {
+			classesToBeLogged.add(fullyQualifiedClassName);
+		}
+	}
+
+	/* package */ void flush() {
```
Comment (albertsteckermeier) [inline report-generator/src/main/java/com/teamscale/report/testwise/jacoco/cache/ClassNotFoundLogger.java]: Missing comment.

File: report-generator/src/main/java/com/teamscale/report/testwise/jacoco/cache/ClassNotFoundLogger.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+		if (classesToBeLogged.isEmpty()) {
+			return;
+		}
+
+		logger.warn(
+				"Found coverage for " + classesToBeLogged
+						.size() + " classes " + " that was not provided. Either you did not provide " +
```
Comment (albertsteckermeier) [inline report-generator/src/main/java/com/teamscale/report/testwise/jacoco/cache/ClassNotFoundLogger.java]: Formatting looks a bit off here.

File: teamscale-gradle-plugin/src/main/kotlin/com/teamscale/TeamscaleUploadTask.kt [hunk 3]
```diff
@@ -59,12 +59,30 @@ open class TeamscaleUploadTask : DefaultTask() {
...
             } catch (e: ConnectException) {
                 throw GradleException("Upload failed (${e.message})", e)
+            } catch (e: SocketTimeoutException) {
+                throw GradleException("Upload failed (${e.message})", e)
             }
         }
     }
 }
+
+fun <T> retry(numOfRetries: Int, block: () -> T): T {
```
Comment (albertsteckermeier) [inline teamscale-gradle-plugin/src/main/kotlin/com/teamscale/TeamscaleUploadTask.kt]: Missing comment.

File: report-generator/src/main/java/com/teamscale/report/testwise/jacoco/cache/ClassNotFoundLogger.java (hunk 2 above)
Comment (DreierF) [reply report-generator/src/main/java/com/teamscale/report/testwise/jacoco/cache/ClassNotFoundLogger.java]: This is how IntelliJ formats it 😕

**Pull request merged**